trieste/models/interfaces.py

```diff
@@ -541,10 +541,12 @@ def __repr__(self) -> str:
     @abstractmethod
     def sample(self, at: TensorType, *, jitter: float = DEFAULTS.JITTER) -> TensorType:
         """
-        :param at: Input points that define the sampler of shape `[N, D]`.
-        :param jitter: The size of the jitter to use when stabilizing the Cholesky
-            decomposition of the covariance matrix.
-        :return: Samples of shape `[sample_size, D]`.
+        :param at: Where to sample the predictive distribution, with shape `[..., 1, D]`, for points
+            of dimension `D`.
+        :param jitter: The size of the jitter to use when stabilising the Cholesky decomposition of
+            the covariance matrix.
+        :return: The samples, of shape `[..., S, 1, L]`, where `S` is the `sample_size` and `L` is
```

Comment:
I'm not sure about this. What if you use a `BatchReparametrizationSampler`? then you would get [..., S, B, L]`, right?

Reply:
Yes, that's a typo. I meant to say B (the key thing being to note that there's an explicit correlated batch dimension).